README.md

```diff
@@ -13,9 +13,9 @@ SDK for generating and publishing events that capture user, application and tech
 
```

Comment:
@robertdumitrescu link and package name in NPM section needs to be updated

Reply:
Will do it under another merge request as per our verbal conversation.